pipelines/forecast_state.py

```diff
@@ -21,21 +21,48 @@
 
 def record_git_info(model_run_dir: Path):
     metadata_file = Path(model_run_dir, "metadata.yaml")
+
+    if metadata_file.exists():
+        with open(metadata_file, "r") as file:
+            metadata = yaml.safe_load(file)
+    else:
+        metadata = {}
+
     try:
         repo = Repository(os.getcwd())
-        branch_name = os.environ.get(
-            "GIT_BRANCH_NAME", Path(repo.head.name).stem
-        )
-        commit_sha = os.environ.get("GIT_COMMIT_SHA", str(repo.head.target))
-    except:
+        branch_name = repo.head.shorthand
+        commit_sha = str(repo.head.target)
+    except Exception as e:
         branch_name = os.environ.get("GIT_BRANCH_NAME", "unknown")
         commit_sha = os.environ.get("GIT_COMMIT_SHA", "unknown")
 
-    metadata = {
+    new_metadata = {
         "branch_name": branch_name,
         "commit_sha": commit_sha,
     }
 
+    metadata.update(new_metadata)
+
+    with open(metadata_file, "w") as file:
+        yaml.dump(metadata, file)
```

Comment:
toml rather than yaml metadata, for consistency with toml elsewhere?

Reply:
Not sure what you mean by "toml elsewhere," but I can change it.

Reply:
https://github.com/CDCgov/pyrenew-hew/pull/251/commits/a31a7e06acbbb6640f5a43c047871faa9f46318d

Reply:
Oh right we used to have some and then moved back to using `.py` files to specify priors